src/plugins/template/src/compiled_model.hpp

```diff
@@ -16,11 +16,11 @@ namespace template_plugin {
 class Plugin;
 class InferRequest;
 
+// ! [compiled_model:header]
 /**
- * @class ExecutableNetwork
- * @brief Interface of executable network
+ * @class CompiledModel
+ * @brief Implementation of compiled model
```

Comment:
<img width="460" alt="Screenshot 2023-03-16 at 1 31 33 PM" src="https://user-images.githubusercontent.com/2566854/225574761-80c33a3f-2dbf-476c-8d2e-5512486794cf.png">
not sure whether it's fixable or not

Reply:
Removed comment from snippets. It is a snippets generation issue.